templates/netlify/remix.config.js

```diff
@@ -1,7 +1,10 @@
 /** @type {import('@remix-run/dev').AppConfig} */
 module.exports = {
   serverBuildTarget: "netlify",
-  server: "./server.js",
+  server:
```

Comment:
This check is to ensure we do not run the Netlify functions server when running `npm run dev`, i.e. `remix dev` as you can't run a custom server via `remix dev`. Without this check, if you run `remix dev` it will error out with the following message:

```
Error: remix dev is not supported for custom servers.
    at Object.dev (/Users/some_user/dev/issues/remix-tailwind/node_modules/@remix-run/dev/cli/commands.js:260:11)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at Object.run (/Users/some_user/dev/issues/remix-tailwind/node_modules/@remix-run/dev/cli/run.js:499:7)
ERROR: "dev:remix" exited with 1.
```